cognite_toolkit/_cdf_tk/cruds/_resource_cruds/configuration.py

```diff
@@ -69,11 +70,14 @@ def get_id(cls, item: SearchConfig | SearchConfigWrite | dict) -> ViewId:
     def dump_id(cls, id: ViewId) -> dict[str, Any]:
         return {"view": id.dump()}
 
+    @classmethod
+    def as_str(cls, id: ViewId) -> str:
```

Comment:
Not going to block this PR on this, but the name and functionality of this method is a little unclear. It creates a directory compatible string from some ID parameter? Perhaps it should have a better name?